docs/sdkx_ios/getting-started.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 sidebar_position: 10
 title: Getting Started
-description: "Getting started with integrating Helpshift in your app."
+description: Follow these simple steps to add Helpshift in-app support to your iOS app right away.
```

Comment:
In inorder the get this as a subtitle I think we should also update line 26. which indicates Getting started with integrating Helpshift in your app. but in Android it is.

Follow these simple steps to add Helpshift in-app support to your
Android app right away